Gemfile

```diff
@@ -43,7 +43,9 @@ group :system_tests do
 
   gem 'beaker-rspec'
   gem 'beaker-windows'
-  gem 'net-ssh'
+  gem 'net-ssh', :git => "https://github.com/net-ssh/net-ssh", :ref => '85ebe31'
+  gem 'ed25519'
```

Comment:
This should be covered by:
https://github.com/voxpupuli/beaker/commit/41b966a87ce5a880f3ff0a9529677dba88347f2e
So you could require `beaker ~> 4.33`